lib/+wbs/@Contacts/Contacts.m

```diff
@@ -234,30 +258,43 @@
         end
 
         function prepare_optimization_matrix(obj)
-            % prepare_optimization_matrix Fills the matrix used in the optimization problem
-
+            % prepare_optimization_matrix Fills the matrix used by the optimization problem solver.
+            % A being the matrix of friction cone constraints, for ...
+            % - quadprog: Ax <= b (should include -x <= 0).
+            % - OSQP    : l <= Ax <= u (should include -Inf <= -x <= 0).
+            %
+            % So in both cases we define: AxLb <= Ax <= AxUb where AxLb = -Inf.
+            
             total_num_vertices = obj.num_vertices * 2; % number of vertex per foot * number feet
-            num_variables = 3 * total_num_vertices; % number of unknowns - 3 forces per vertex
+            num_variables = 3 * total_num_vertices; % number of unknowns (3 force components per vertex)
             num_constr = 5 * total_num_vertices; % number of constraint: simplified friction cone + non negativity of vertical force
             % fill the optimization matrix
             obj.A = zeros(num_constr, num_variables);
-            obj.b = zeros(num_constr, 1);
+            obj.AxUb = zeros(num_constr, 1);
+            obj.AxLb = -Inf(num_constr, 1);
```

Comment:
I would substitute -inf with something like -1e^17 to avoid strange MatLab interpretation of inf

Reply:
Can you be more specific?

Reply:
I would have 
`obj.AxLb = -1e17*ones(num_constr, 1);`
rather than inf, to avoid strange matlab interpretation of inf from matlab 

This is anyway a minor comment

Reply:
My question was concerning your comment "strange matlab interpretation of inf from matlab"?

Reply:
As agree in our F2F discussion, there is no observed issue w.r.t. the use of `Inf`. It's a personal choice in some cases to use  `1e17` instead. `Inf` is unambiguous so we'll leave it that way.